polyhost/services/log_bundle.py

```
@@ -399,6 +514,15 @@ def environment_text(include_slow: bool = False) -> str:
         autostart = autostart_detail()
         if autostart:
             lines.append(f"Autostart        : {autostart}")
+    # The crash markers are the first thing worth knowing on a "the app
+    # vanished" report, and the body is read long before the attachment is
+    # opened. Never fatal: diagnostics must not be able to break a bug report.
+    try:
+        crashes = crash_summary()
+        if crashes:
+            lines.append(f"Crash log        : {crashes}")
+    except Exception:  # noqa: BLE001 — diagnostics must never break the bundle
+        pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)